### DISCUSSION TITLE
Alpha 0.4.1

### DISCUSSION THREAD

**Discussion by iLynxcat** [spacedriveapp/spacedrive#2640] [Announcements]
This minor patch adds new app shortcuts and improvements to the Quick Preview feature, as well as fixing various bugs.

## 🪐 Core Updates
* Updated terminal log messages to a prettier, easier-to-read format - by [@matheus-consoli](https://github.com/matheus-consoli) in [#2476](https://github.com/spacedriveapp/spacedrive/pull/2476)

### 🖥️ Desktop
* Changed the sidebar toggle shortcut to <kbd>⌘</kbd><kbd>S</kbd> (macOS) / <kbd>Ctrl</kbd><kbd>S</kbd> (Windows/Linux) - by [@ameer2468](https://github.com/ameer2468) in [#2625](https://github.com/spacedriveapp/spacedrive/pull/2625)
* Added zoom shortcuts <kbd>⌘</kbd><kbd>+ / -</kbd> (macOS) / <kbd>Ctrl</kbd><kbd>+ / -</kbd> (Windows/Linux) and pinch-to-zoom in Quick Preview - by [@myung03](https://github.com/myung03) in [#2635](https://github.com/spacedriveapp/spacedrive/pull/2635)

### 🖥️ Desktop
* Fixed a bug where right-click could open the native context menu instead of the Spacedrive context menu - by [@myung03](https://github.com/myung03) in [#2637](https://github.com/spacedriveapp/spacedrive/pull/2637)

## 🗞️ General/Development Updates
* Translated the Top Bar/Overview to Russian and Belarusian - by [@artsiom-voitas](https://github.com/artsiom-voitas) in [#2628](https://github.com/spacedriveapp/spacedrive/pull/2628)
* Updated the Github bug report template - by [@iLynxcat](https://github.com/iLynxcat) in [#2622](https://github.com/spacedriveapp/spacedrive/pull/2622)
* Updated the `rspc-tauri` and `tailwindcss-animate` dependencies - by [@Rocky43007](https://github.com/Rocky43007) ([#2634](https://github.com/spacedriveapp/spacedrive/pull/2634)) and [@iLynxcat](https://github.com/iLynxcat) ([#2631](https://github.com/spacedriveapp/spacedrive/pull/2631))
* Updated the `openssl-dev` dependency to `openssh-devel` for Fedora (39+) Linux developers - by [@greendoescode](https://github.com/greendoescode) in [#2630](https://github.com/spacedriveapp/spacedrive/pull/2630)
* Fixed an issue with checking out Pull Requests from forks - by [@iLynxcat](https://github.com/iLynxcat) in [#2631](https://github.com/spacedriveapp/spacedrive/pull/2631)

### 📱 Mobile (soon)<sup>tm</sup>
* Improved Media View thumbnails - by [@ameer2468](https://github.com/ameer2468) in [#2626](https://github.com/spacedriveapp/spacedrive/pull/2626)
* Improved the sync settings UI/UX - by [@ameer2468](https://github.com/ameer2468) in [#2636](https://github.com/spacedriveapp/spacedrive/pull/2636)

## New Contributors
* [@greendoescode](https://github.com/greendoescode) made their first contribution in [#2630](https://github.com/spacedriveapp/spacedrive/pull/2630)

**Full Changelog**: [https://github.com/spacedriveapp/spacedrive/compare/0.4.0...0.4.1](https://github.com/spacedriveapp/spacedrive/compare/0.4.0...0.4.1)

<hr /><em>This discussion was created from the release <a href='https://github.com/spacedriveapp/spacedrive/releases/tag/0.4.1'>Alpha 0.4.1</a>.</em>

**Comment by artem-8178** [comment]
Do you plan to add appimage as an installation option?

**Reply by greendoescode** [reply]
~~I'm not quite sure about appimages, I'll have a look for ya. It shouldn't be _that_ hard to add haha~~ Good reason it's not added listed below. 
I'm currently working on adding flatpak support though, if that's any interest to you.

**Reply by HeavenVolkoff** [reply]
AppImage support is not planned. We used to have an AppImage official release, but it had way too many exclusive problems for us to keep maintaining. Flatpak, PKGBUILD (ArchLinux) and RPM (Fedora) support are planned though

**Reply by artem-8178** [reply]
flatpak is also a good option
